augly/image/intensity.py

```diff
@@ -286,6 +285,8 @@ def sharpen_intensity(factor: float, **kwargs) -> float:
 def shuffle_pixels_intensity(factor: float, **kwargs) -> float:
     return factor * 100.0
 
+def skew_intensity(level: float, **kwargs) -> float:
+    return min(level, 100.0)
```

Comment:
This is how @jbitton  meant to change the intensity function definition:
```suggestion
    max_level = 2.0
    return min((abs(level) / max_level) * 100.0, 100.0)
```